migrate.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
```

Comment:
Generally we want to enable additional options to catch certain errors early:

```
set -oeu pipefail
```

Reply:
I am not an expert on bash scripting, so I will take your word on this being a good practice!